packages/pyodide-kernel-extension/src/index.ts

```diff
@@ -13,6 +13,7 @@ import { IKernel, IKernelSpecs } from '@jupyterlite/kernel';
 import { IBroadcastChannelWrapper } from '@jupyterlite/contents';
 
 export * as KERNEL_SETTINGS_SCHEMA from '../schema/kernel.v0.schema.json';
+import * as KERNEL_ICON from '!!url-loader!../style/img/pyodide.svg';
```

Comment:
Normally we should be able to do without `url-loader` and use the webpack 5 Asset Module instead.

Reply:
For example the p5 kernel brings its own logo here: https://github.com/jupyterlite/p5-kernel/blob/fbda9a78adde48e7ed9012886c28f68c81e9270d/packages/p5-kernel-extension/src/index.ts#L15

Although it's a png.

Reply:
sure, asset loader is rad.

Reply:
it's "working" e0457fa